sdk/Lusid.Sdk/Utilities/LusidApiFactory.cs

```diff
@@ -76,6 +76,9 @@ public LusidApiFactory(Configuration configuration)
                     throw new Exception($"Unable to create type {api}");
                 }
 
+                // Replace the default implementation of the ExceptionFactory with a custom one defined by FINBOURNE
+                impl.ExceptionFactory = LusidExceptionHandler.CustomExceptionFactory;
```

Comment:
Do we want to test that APIs that are called indeed have their default exception factory overridden?

Reply:
Good idea. Have added `ExceptionFactoryIsOverriddenWithCustomImplementation`